.github/workflows/test-env-deploy.yml

```diff
@@ -213,7 +239,17 @@ jobs:
       - name: Install Dependencies
         if: steps.cache-node-modules.outputs.cache-hit != 'true'
         run: NODE_OPTIONS=--max_old_space_size=4096 npm install
-
+      - name: Testmo reports threads init
+        run: | 
+            npx testmo automation:run:submit-thread \
+            --instance "$TESTMO_URL" \
+            --run-id "${{ needs.test-setup.outputs.testmo-run-id }}" \
+            --results results/*.xml
+        env:
+          CI_TOTAL: ${{ matrix }}
+          CI_INDEX: ${{ steps.get_containers.outputs.containerIndexes }}
```

Comment:
Can you add YAML comments explaining what those vars do? I can't find docs about them, I guess it's concurrency management

Reply:
Removed